### PR TITLE
2 packages from monaqa/slydifi at 0.4.0

### PR DESCRIPTION
**Pull request by monaqa**
A SATySFi document class for creating presentations slides

This pull-request concerns:
-`satysfi-class-slydifi.0.4.0`
-`satysfi-class-slydifi-doc.0.4.0`

---
* Homepage: https://github.com/monaqa/slydifi
* Source repo: git+https://github.com/monaqa/slydifi.git
* Bug tracker: https://github.com/monaqa/slydifi/issues

---
:camel: Pull-request generated by opam-publish v2.0.3
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- [x] Add to snapshot `snapshot-develop` :: satysfi-class-slydifi-doc.0.4.0 satysfi-class-slydifi.0.4.0
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-5`~~ (Inconsistent)
- [x] Add to snapshot `snapshot-stable-0-0-6` :: satysfi-class-slydifi-doc.0.4.0 satysfi-class-slydifi.0.4.0

### CODE REVIEW

File: packages/satysfi-class-slydifi/satysfi-class-slydifi.0.4.0/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+bug-reports: "https://github.com/monaqa/slydifi/issues"
+dev-repo: "git+https://github.com/monaqa/slydifi.git"
+
+depends: [
+  "satysfi" {>= "0.0.6" & < "0.0.7"}
+  "satyrographos" {>= "0.0.2" & < "0.0.3"}
+  "satysfi-enumitem" {>= "2.0.0" & < "3.0.0"}
+  "satysfi-figbox" {>= "0.1.3"}
+  "satysfi-base" {>= "1.0.0"}
+  "satysfi-railway" {>= "0.1.0"}
```
Comment (na4zagin3) [inline packages/satysfi-class-slydifi/satysfi-class-slydifi.0.4.0/opam:18]: Can you specify the upper bounds like these?

```suggestion
  "satysfi-figbox" {>= "0.1.3" & < "0.2.0"}}
  "satysfi-base" {>= "1.0.0" & < "2.0.0"}
  "satysfi-railway" {>= "0.1.0" & < "0.2.0"}
```

File: packages/satysfi-class-slydifi-doc/satysfi-class-slydifi-doc.0.4.0/opam [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+license: "MIT"
+homepage: "https://github.com/monaqa/slydifi"
+bug-reports: "https://github.com/monaqa/slydifi/issues"
+dev-repo: "git+https://github.com/monaqa/slydifi.git"
+
+depends: [
+  "satysfi" {>= "0.0.6" & < "0.0.7"}
+  "satyrographos" {>= "0.0.2" & < "0.0.3"}
+  "satysfi-class-slydifi" {= "%{version}%"}
+  "satysfi-easytable" {>= "1.0.0"}
```
Comment (na4zagin3) [inline packages/satysfi-class-slydifi-doc/satysfi-class-slydifi-doc.0.4.0/opam:16]: Can you specify the upper bound?

```suggestion
  "satysfi-easytable" {>= "1.0.0" & < "2.0.0"}
```

**Review by na4zagin3 — CHANGES_REQUESTED** [review]
Thank you for sending the new version of slydifi!

Can you please add upper bounds to the version constraints?

**Comment by monaqa** [comment]
Thank you for your review. Fixed.

**Comment by na4zagin3** [comment]
Let me rerun the CI by reopening the ticket.

**Review by na4zagin3 — APPROVED** [review]

**Comment by na4zagin3** [comment]
Thank you!

**Pull request merged**